test/integration/src/py/test_utilities.py

```diff
@@ -215,20 +230,22 @@ def get_password(network_definition_file_json):
 
 
 def get_eth_balance(transfer_request: EthereumToSifchainTransferRequest):
-    network_element = f"--ethereum_network {transfer_request.ethereum_network} " if transfer_request.ethereum_network else ""
-    symbol_element = f"--symbol {transfer_request.ethereum_symbol} " if transfer_request.ethereum_symbol else ""
-    private_element = f"--ethereum_private_key_env_var \"{transfer_request.ethereum_private_key_env_var}\"" if transfer_request.ethereum_private_key_env_var else ""
-    command_line = " ".join(
-        [f"yarn -s --cwd {transfer_request.smart_contracts_dir}",
-         f"integrationtest:getTokenBalance",
-         f"--ethereum_address {transfer_request.ethereum_address}",
-         f"--json_path {transfer_request.solidity_json_path}",
-         private_element,
-         symbol_element,
-         network_element]
-    )
-    result = run_yarn_command(command_line)
-    return int(result["balanceWei"])
+    w3 = Web3(Web3.WebsocketProvider(ethereum_websocket))
+    return w3.eth.get_balance()
```

Comment:
This probably won't work - the original method gets the balance of the account defined in transfer_request.ethereum_symbol, and the new one gets the account balance of the first key that w3.eth knows about (I think?)

Reply:
This was one of those things we need to refactor, it gets the balance of the first key as you stated rather then a specified key. Does seem to work for that key though.